### ISSUE TITLE
Sum of archive deduplicated sizes does not match all-archives deduplicated size

### ISSUE THREAD

**Issue by johnp789** [borgbackup/borg#1532]
My borg repo has six archives.  If their info is listed in turn, the sum of the six deduplicated sizes only add up to about 50GB, not 193.35GB shown for the all-archives total.    It seems that they should be close to equal.  `du` shows a total in line with the larger all-archives total.  

I'm worried that there is something inconsistent in my repo.  

```
$ uname -a
Linux storm 4.6.4-1-ARCH #1 SMP PREEMPT Mon Jul 11 19:12:32 CEST 2016 x86_64 GNU/Linux
$ borg --version
borg 1.0.6
$ borg list /run/media/john/backup3/borg
john_Documents                       Fri, 2016-03-25 21:01:44
john_Pictures-1458959201             Fri, 2016-03-25 21:26:41
john_Documents_2016-06-05            Sun, 2016-06-05 20:59:13
john_Pictures-1467563569             Sun, 2016-07-03 11:32:49
john_Documents-2016-08-21            Sun, 2016-08-21 19:58:23
RasPi2_Arch_2016-08-22               Tue, 2016-08-23 19:40:14
$ borg list /run/media/john/backup3/borg | cut -f 1 -d ' ' | xargs -I '{}' -n 1 borg info /run/media/john/backup3/borg::'{}'        
Name: john_Documents
Fingerprint: 2287af53b4334fcee9cdaca3a23ff40c4a8600af940641d6f2f07d256c0afd7e
Hostname: storm
Username: john
Time (start): Fri, 2016-03-25 21:01:44
Time (end):   Fri, 2016-03-25 21:12:23
Command line: borg create -v --stats --progress --compression=lz4 /run/media/john/backup3/borg::john_Documents /mnt/Hitachi_1.5TB/ho
me/john/Documents --exclude *.o --exclude /mnt/Hitachi_1.5TB/home/john/Documents/scratch
Number of files: 44058

                       Original size      Compressed size    Deduplicated size
This archive:               13.23 GB              9.78 GB            224.32 kB
All archives:              362.94 GB            350.08 GB            193.35 GB

                       Unique chunks         Total chunks
Chunk index:                  166947               367646
Name: john_Pictures-1458959201
Fingerprint: 43c7315341c4dac921f845e7eb7c4af42256848a5e2aedaeb8cb0a405a74f96b
Hostname: storm
Username: john
Time (start): Fri, 2016-03-25 21:26:41
Time (end):   Fri, 2016-03-25 23:30:51
Command line: borg create -v --stats --progress --compression=lz4 /run/media/john/backup3/borg::john_Pictures-{now:%s} /mnt/Hitachi_
1.5TB/home/john/Pictures/
Number of files: 51686

                       Original size      Compressed size    Deduplicated size
This archive:              135.63 GB            135.09 GB              2.53 MB
All archives:              362.94 GB            350.08 GB            193.35 GB

                       Unique chunks         Total chunks
Chunk index:                  166947               367646
Name: john_Documents_2016-06-05
Fingerprint: fc716c397ae3e8d43c352c2e79a6c4a4fb29c9fbad0331571b2b6206d002d871
Hostname: storm
Username: john
Time (start): Sun, 2016-06-05 20:59:13
Time (end):   Sun, 2016-06-05 20:59:29
Command line: borg create -v --stats --progress --compression=lz4 /run/media/john/backup3/borg::john_Documents_2016-06-05 /mnt/Hitac
hi_1.5TB/home/john/Documents --exclude *.o --exclude /mnt/Hitachi_1.5TB/home/john/Documents/scratch
Number of files: 44134

                       Original size      Compressed size    Deduplicated size
This archive:               13.26 GB              9.80 GB            229.57 kB
All archives:              362.94 GB            350.08 GB            193.35 GB

                       Unique chunks         Total chunks
Chunk index:                  166947               367646
Name: john_Pictures-1467563569
Fingerprint: cb4bf18df9ca76853c4dc9e6b396cbb840c20a3e848e1e2deeba8e37b0b81697
Hostname: storm
Username: john
Time (start): Sun, 2016-07-03 11:32:49
Time (end):   Sun, 2016-07-03 11:49:54
Command line: borg create -v --stats --progress --compression=lz4 /run/media/john/backup3/borg::john_Pictures-{now:%s} /mnt/Hitachi_
1.5TB/home/john/Pictures/
Number of files: 56319

                       Original size      Compressed size    Deduplicated size
This archive:              153.85 GB            153.25 GB             18.13 GB
All archives:              362.94 GB            350.08 GB            193.35 GB

                       Unique chunks         Total chunks
Chunk index:                  166947               367646
Name: john_Documents-2016-08-21
Fingerprint: 55da72a9b5fe5b5d320b8f55b1ca6818d3665dc216c98e2a3a13d1c234bd3606
Hostname: storm
Username: john
Time (start): Sun, 2016-08-21 19:58:23
Time (end):   Sun, 2016-08-21 20:23:24
Command line: /usr/bin/borg create -v --stats --progress --compression=lz4 /run/media/john/backup3/borg::john_Documents-{now:%Y-%m-%
d} /mnt/Hitachi_1.5TB/home/john/Documents --exclude *.o --exclude /mnt/Hitachi_1.5TB/home/john/Documents/scratch
Number of files: 44241

                       Original size      Compressed size    Deduplicated size
This archive:               45.71 GB             40.89 GB             31.09 GB
All archives:              362.94 GB            350.08 GB            193.35 GB

                       Unique chunks         Total chunks
Chunk index:                  166947               367646
Name: RasPi2_Arch_2016-08-22
Fingerprint: 7040d987f3d51dcfb65fc573f2675ec1a9abe94133e42e8596eae1fcddf148e7
Hostname: storm
Username: john
Time (start): Tue, 2016-08-23 19:40:14
Time (end):   Tue, 2016-08-23 19:41:25
Command line: /usr/bin/borg create -v --stats --progress --compression=lz4 /run/media/john/backup3/borg::RasPi2_Arch_2016-08-22 /mnt
/Hitachi_1.5TB/home/john/machine_image/RasPi2_Arch_2016-08-22
Number of files: 6

                       Original size      Compressed size    Deduplicated size
This archive:                1.27 GB              1.27 GB              1.27 GB
All archives:              362.94 GB            350.08 GB            193.35 GB

                       Unique chunks         Total chunks
Chunk index:                  166947               367646
$ du -sh /run/media/john/backup3/borg/
181G    /run/media/john/backup3/borg/
```

**Comment by enkore** [comment]
This is correct. The deduplicated size of all archives doesn't add up to the deduplicated size of the repo ("all archives"), because the two are meaning different things here (which is confusing, yes).

This archive / deduplicated size = amount of data stored **only** for this archive (=unique chunks)
All archives / deduplicated size = amount of data stored in the repo

**Comment by johnp789** [comment]
Thanks---the two-line summary above makes it totally clear.

**Comment by ThomasWaldmann** [comment]
fixed by #1566
